docs/usage.md

```diff
@@ -3,6 +3,7 @@
 <!-- TOC -->
 
 - [Intro](#intro)
+  - [`error`](#error)
```

Comment:
toc issue

Reply:
df69abc Fix TOC & job states image in usage docs